### PR TITLE
Update system-requirements.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #2698 - Updated the document with the reason for NAS support limitation.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfBServer2019/plan/system-requirements.md [hunk 1]
```diff
@@ -538,7 +538,7 @@ Skype for Business Server 2019 can use the same file share for all file storage.
 - We recommend a shared cluster for the file share. If you're using one, you should cluster Windows Server 2012 or Windows Server 2012 R2. Why the latest Windows? Older versions may not have the right permissions to enable all features. You can use Cluster Administrator to create the file shares, and this [Creating a Cluster](https://support.microsoft.com/en-us/help/224967) KB article will help you with those details.
     
 > [!CAUTION]
-> You should know that using network attached storage (NAS) as a file share isn't supported, so use one of the options listed above. 
+> You should know that using network attached storage (NAS) as a file share isn't supported, so use one of the options listed above. This support limitation is caused by the variable design of NAS devices that have to provide file system adaptability to the Windows Server-based computer that accesses the devices' shared file system.
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/system-requirements.md]:
```suggestion
> You should know that using network attached storage (NAS) as a file share isn't supported, so use one of the options previously listed. This support limitation is caused by the variable design of NAS devices that have to provide file system adaptability to the Windows Server-based computer that accesses the devices' shared file system.
```

Comment (sowjanya365) [reply Skype/SfBServer2019/plan/system-requirements.md]: Thank you for the review. Suggested changes are applied. Kindly verify. Thank you.

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Skype/SfBServer2019/plan/system-requirements.md (hunk 1 above)
Comment (MicrosoftHeidi) [reply Skype/SfBServer2019/plan/system-requirements.md]: We can accept the additional explanation for why there is a support limitation, but the next change doesn't take voice into consideration, we should leave listed above.

**Pull request merged**